src/main/java/com/manardenza/dao/HotelDaoImpl.java

```diff
@@ -1,14 +1,20 @@
 package com.manardenza.dao;
 
 import com.manardenza.entity.Hotel;
+import com.manardenza.utils.DatabaseSeeder;
 
+import java.io.File;
+import java.util.ArrayList;
 import java.util.List;
-import java.util.stream.Collectors;
 
 public class HotelDaoImpl extends AbstractDao<Hotel> {
 
     private static HotelDaoImpl instance;
 
+    {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Move the contents of this initializer to a standard constructor or to field initializers. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1171)